lib/message_bus/backends/redis.rb

```diff
@@ -167,13 +177,6 @@ def ensure_backlog_flushed
 
         @in_memory_backlog.delete_at(0) unless try_again
       end
-
-      if try_again
-        sleep 0.005
-        # in case we are not connected to the correct server
```

Comment:
why dod we get rid of the sleep here, don't want us hitting redis really hard during readonly